### PR TITLE
Fabric Website RFC

### PR DESCRIPTION
**Pull request by erin-hughes**
Signed-off-by: Erin Hughes <Erin.Hughes@ibm.com>

### CODE REVIEW

**Comment by mbwhite** [comment]
+1 here

File: text/0000-fabric-website.md
Comment (davidkhala) [inline text/0000-fabric-website.md:17]: The link page loading time on my first access via browser is more than I expect. A quick analysis shows the pending of wss://warm-reaches-04614.herokuapp.com/sockjs-node last for around 50 seconds.

**Comment by davidkhala** [comment]
And is there a plan to contribute this web project source code under Hyperledger?

**Comment by mbwhite** [comment]
@davidkhala the reference section details the location in the Hyperledger project it will be deployed once the RFC is approved

File: text/0000-fabric-website.md
Comment (yacovm) [reply text/0000-fabric-website.md:17]: That's because Heroku (i guess it's free tier?) puts inactive applications to sleep, and wakes them up when traffic is incoming. It has nothing to do with the site itself, I think.

File: text/0000-fabric-website.md
Comment (yacovm) [inline text/0000-fabric-website.md:52]: Since it is github pages, it's not going to be on the same repository as the Fabric core (https://github.com/hyperledger/fabric), right? 

Can we make sure we limit the Fabric core repository to its intended role, to be the Fabric core? :-)

File: text/0000-fabric-website.md
Comment (yacovm) [inline text/0000-fabric-website.md:15]: This is a very low level comment, but, can we perhaps make the [header](https://warm-reaches-04614.herokuapp.com/static/media/Fabric%20website%20header.dd2503aa.png) be blue instead of red? Red is a very opinionated color that represents several nationalities and certain ideologies while blue is rather neutral.

File: text/0000-fabric-website.md
Comment (mbwhite) [reply text/0000-fabric-website.md:52]: @yacovm  a gh-pages site on the URL  `hyperledger.gituhub.io/fabric-website` can only be served from the `fabric-website` repository.

File: text/0000-fabric-website.md
Comment (erin-hughes) [reply text/0000-fabric-website.md:17]: Yes that's right, we're hosting on the free version of Heroku so the load time will be quite long if the site hasn't been active in a while. We won't see this when we deploy to gh-pages.

File: text/0000-fabric-website.md
Comment (erin-hughes) [reply text/0000-fabric-website.md:15]: The reason that we went with the red was to tie in with the [Fabric logo](https://www.hyperledger.org/wp-content/uploads/2018/04/fabric-logo.png), which also incorporates red.

File: text/0000-fabric-website.md
Comment (ryjones) [reply text/0000-fabric-website.md:52]: https://github.com/hyperledger/fabric-website is served via [CNAME](https://github.com/hyperledger/fabric-website/blob/gh-pages/CNAME) as [fabric.hyperledger.org](https://fabric.hyperledger.org)

**Comment by jmason90** [comment]
i> the site is useful IF the audience is an operator or a developer and already knows what Fabric is and how they may want to use it. Other than that, I don't see it as "high value". I would not use a site like this to provide an overview of Fabric or how it can be used.

i> I avoid web sites with tons of white space ("low content sites"). They may work for marketing ( selling a Tesla etc ) but I believe most of the audience is technical and would prefer a "high content, clean, logical layout" with minimal scrolling

i> the documentation link should point to latest by default - https://hyperledger-fabric.readthedocs.io/en/latest/

i> chaincode development 
I'll never be a fan of using the term chaincode over smart contract. It puts off everyone who is new to Fabric

**Review by denyeart — APPROVED** [review]
Overall I'm in favor of proceeding. Jim's comments make sense and can be incorporated into the actual website.

**Comment by ryjones** [comment]
Hi - is this approved, or not? It's been a month. @hyperledger/fabric-core-maintainers

File: text/0000-fabric-website.md
Comment (SamYuan1990) [inline text/0000-fabric-website.md:22]: question here for performance testing, see I am currently developing/maintaining some project related with fabric performance such as
https://github.com/guoger/tape
and 
https://github.com/SamYuan1990/Probe
so
what's the plan as generic guide for advance knowledges. such as performance testing, HA, etc...
is there any plan for performance testing specific?
is there any plan for link with other fabric labs project as mini fab?

Comment (davidkhala) [reply text/0000-fabric-website.md:22]: **additional content on other relevant topics** is a good start for us to brainstorm, Can this Fabric resources navigation/indexing Website provides more opportunity for light-weight, third-party community projects in Fabric ecosystem?

File: text/0000-fabric-website.md
Comment (caod123) [inline text/0000-fabric-website.md:24]: Regarding the getting started under the developer page, would it make more sense to link to the wiki page for installing necessary development tools/prerequisites instead of linking to the IBM VS Code extension? Having a link to the extension is probably fine, but not everyone will intend on using the VS Code extension for setting up their development environment.

File: text/0000-fabric-website.md
Comment (caod123) [inline text/0000-fabric-website.md:42]: Rather than linking to the README and pointing people to the other links in the Community section of the README, maybe the user experience here would feel better to just have additional links directly from this site to the more relevant community links.

**Review by pamandrejko — COMMENTED** [review]
I really like the new illustrations. 

I noticed the prototype site points to Fabric v2.1 (latest). We are up to Fabric v2.3 now and v2.2 is the LTSR. Should default to the LTSR or the latest release? I assume when this goes live we would add links to those other versions?

**Review by mbwhite — APPROVED** [review]

**Review by awjh-ibm — APPROVED** [review]

**Review by andrew-coleman — APPROVED** [review]

**Comment by bestbeforetoday** [comment]
> I'll never be a fan of using the term chaincode over smart contract. It puts off everyone who is new to Fabric

I also thought Smart Contract was the preferred term. I think it is what is used in the majority of the existing developer-focused documentation.

Also on the current **Developer** page, I don't think "Gateway SDK development" is accurate. I think what developers are interested in doing is building client applications (for which they will make use of the client / gateway SDKs), not developing the SDKs themselves.

But these are just comments on the current draft content and should be addressed as part of implementing the site. The RFC proposal itself looks good to me. 👍

**Review by bestbeforetoday — APPROVED** [review]

**Review by jt-nti — APPROVED** [review]

**Comment by denyeart** [comment]
Ok, with the set of approvals I'll go ahead and merge.
The remaining comments were mostly about the actual website content, which can be iterated on through the normal PR process in fabric-website repository.

**Pull request merged**
